kafka/source/config/300-kafkabinding.yaml

```diff
@@ -22,6 +22,15 @@ metadata:
   name: kafkabindings.bindings.knative.dev
 spec:
   group: bindings.knative.dev
+  preserveUnknownFields: false
```

Comment:
oh... look, I did forget that part in here: https://github.com/knative/eventing-contrib/commit/84c21fbb9cc0d33fe5ae24e6e2593b64cc11f0e6   - sorry, @aliok